msf_updates/src/position_msf/init_yaw.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+PACKAGE = 'msf_updates'
+import roslib;roslib.load_manifest(PACKAGE)
+import rospy
+from geometry_msgs.msg import Vector3
+
+import dynamic_reconfigure.client
+
+client = dynamic_reconfigure.client.Client("msf_gps_pose_estimator/position_sensor", timeout=30)
+
+# Write init_yaw MSF parameter from orientation message
+def callback_orientation(data):
+    client.update_configuration({"position_yaw_init":data.z})
+
+if __name__ == "__main__":
+    rospy.init_node("dynamic_client")
+
```

Comment:
https://www.python.org/dev/peps/pep-0008/
generally, if you write python code, I would recommend using pylint: http://www.pylint.org/ (can be used and installed as plugins in most editors, such as sublime, atom, eclipse...)